bee-common/bee-packable-derive/src/field_info.rs

```diff
@@ -40,20 +41,35 @@ impl FieldInfo {
 
         let ident = format_ident!("field_{}", index);
 
+        let mut unpack_error_with_opt = None;
+        let mut verify_with_opt = None;
+
         for attr in filter_attrs(&field.attrs) {
-            if let Some(unpack_error_with) =
-                attr.parse_args_with(|stream: ParseStream| parse_kv("unpack_error_with", stream))?
-            {
-                return Ok(Self {
-                    unpack_error_with,
-                    ident,
-                    pattern_ident,
-                    ty: field.ty.clone(),
-                });
+            if let Some(verify_with) = attr.parse_args_with(|stream: ParseStream| {
+                let opt = parse_kv("verify_with", stream)?;
+                if opt.is_none() {
+                    skip_stream(stream)?;
+                }
+                Ok(opt)
+            })? {
+                verify_with_opt = Some(verify_with);
+            }
+
+            if let Some(unpack_error_with) = attr.parse_args_with(|stream: ParseStream| {
+                let opt = parse_kv("unpack_error_with", stream)?;
+                if opt.is_none() {
+                    skip_stream(stream)?;
+                }
+                Ok(opt)
+            })? {
+                unpack_error_with_opt = Some(unpack_error_with);
             }
         }
+
         Ok(Self {
-            unpack_error_with: default_unpack_error_with.clone(),
+            unpack_error_with: unpack_error_with_opt.unwrap_or_else(|| default_unpack_error_with.clone()),
+            verify_with: verify_with_opt
+                .unwrap_or_else(|| parse2(quote!(|_| -> Result<(), Self::UnpackError> { Ok(()) })).unwrap()),
```

Comment:
`parse2` and `quote!` do a non trivial amount of work. I could add a new parameter to this function so it receives a `default_verify_with` so we only compute it once given that it would be the same everywhere.

Reply:
Oh, I forgot about the lazy aspect of `unwrap_or_else`. Considering that I think it's fine as it is!